src/utils/getPriceGuidance.ts

```diff
@@ -0,0 +1,66 @@
+import { flatMap, reduce } from "lodash"
+import metaphysics from "../lib/metaphysics"
+
+export const getPriceInDollars = priceCents => {
+  if (priceCents && priceCents.min) {
+    return priceCents.min / 100
+  } else if (priceCents && priceCents.max) {
+    return priceCents.max / 100
+  } else {
+    return null
+  }
+}
+
+export const getPriceGuidance = async (slug: string) => {
+  const results: any = await metaphysics(`{
+    marketingCollection(slug: "${slug}") {
+      artworks(
+        size: 5,
+        sort: "prices",
+        price_range: "10-*"
+      ) { 
+        hits {
+          price
+          priceCents {
```

Comment:
@xtina-starr Not to add more to this convo but shouldn't we also be getting `price_currency` so we can do the conversion to `USD` if needed?

Reply:
I was thinking that the price_cents was already converted in [gravity](https://github.com/artsy/gravity/blob/4a7d28b5954e1c52e44bc1522f76ca621b535517/app/models/util/for_sale.rb#L202-L213)

Reply:
That doesn't do any exchange rate conversion, that just handles 'dollars to cents' in terms of, knowing to convert the inputted price (in 'dollars' or major units) to the minor unit, usually by multiplying by either 100, or 1 (Japanese yen, etc.).

I _think_ that this PR is overall still not really fully currency aware. There's a hardcoded `/ 100` in there. Right now this works since all `acquireable` works are in USD, but this has the potential to be averaging works that are priced in different currencies. The `price_cents` field in Gravity is not in USD, but in whatever the local currency selected.

For this to be able to fully calculate a base USD price by averaging artworks in the collection, it would need to be able to properly convert an artwork priced in JPY, to an equivalent USD (using a current exchange rate). I don't think we actually do this anywhere besides Looker, and Gravity, currently! So there's no existing pattern for that (but we can add exchange rates to Metaphysics) to make those available. Worth keeping in mind for the future (and maybe adding a comment/documenting somewhere)?